browse/routes/ui.py

```diff
@@ -164,25 +287,68 @@ def show_email(show_email_hash: str, arxiv_id: str) -> Response:
 # Maybe auth protected URL in arxiv-browse?
 # ('will the auth service allow paths not defined in it's
 #  repo to be protected?')
-@blueprint.route('/auth/show-endorsers/<path:arxiv_id>')
+@blueprint.route('auth/show-endorsers/<path:arxiv_id>')
 def show_endorsers(arxiv_id: str) -> Response:
     """Show endorsers for an article."""
     raise InternalServerError(f'Not yet implemented {arxiv_id}')
 
 
-@blueprint.route('/refs/<path:arxiv_id>')
+@blueprint.route('refs/<path:arxiv_id>')
 def refs(arxiv_id: str) -> Response:
     """Show the references for an article."""
     raise InternalServerError(f'Not yet implemented {arxiv_id}')
 
 
-@blueprint.route('/cits/<path:arxiv_id>')
+@blueprint.route('cits/<path:arxiv_id>')
 def cits(arxiv_id: str) -> Response:
     """Show the citations for an artcile."""
     raise InternalServerError(f'Not yet implemented {arxiv_id}')
 
 
-@blueprint.route('/form')
+@blueprint.route('form')
 def form(arxiv_id: str) -> Response:
     """Old form interface to lists of articles."""
     raise InternalServerError(f'Not yet implemented {arxiv_id}')
+
+
+@blueprint.route('archive/', defaults={'archive': None})
+@blueprint.route('archive/<archive>')
+def archive(archive: str):  # type: ignore
+    """Landing page for an archive."""
+    response, code, headers = archive_page.get_archive(archive)  # type: ignore
+    if code == status.HTTP_200_OK or code == status.HTTP_404_NOT_FOUND:
+        return render_template(response['template'], **response), code, headers
+    elif code == status.HTTP_301_MOVED_PERMANENTLY:
+        return redirect(headers['Location'], code=code)
+    elif code == status.HTTP_304_NOT_MODIFIED:
+        return '', code, headers
+    else:
+        return response, code, headers
```

Comment:
Minor nit: dropping this out of the ``else`` block makes it clearer that this is the catch-call path. Looks like you guys are doing this in some cases, e.g. L338, below.